warehouse/manage/forms.py

```diff
@@ -382,6 +382,18 @@ def validate_macaroon_id(self, field):
 # /manage/organizations/ forms
 
 
+class OrganizationActivateBillingForm(wtforms.Form):
+    terms_of_service_agreement = wtforms.BooleanField(
+        validators=[
+            wtforms.validators.AnyOf(
+                [True],
+                message="Terms of Service must be accepted.",
+            ),
```

Comment:
I _think_ this can be:

```suggestion
            wtforms.validators.DataRequired(
                message="Terms of Service must be accepted.",
            ),
```

Because the valid values for a `BooleanField` are truthy/falsey

Reply:
https://github.com/pypi/warehouse/pull/17376/commits/c7694bb839be795d0f55e8f4870cfdc3479aac0a